beheer.md

```diff
@@ -22,6 +22,40 @@ In dit document worden allerlei procedures en overzichten beschreven die van bel
 ## Lijst van deelnemende partijen en personen
 Bij de ontwikkeling van Open API standaarden nemen diverse partijen deel. Het is handig voor ontwikkelaars maar ook voor beheerders om daar een overzicht van te hebben. Daarom dient de project manager of scrum master bij de aanvang van een ontwikkelproject maar ook tijdens het project als de teamsamenstelling wijzigt de [lijst met Deelnemende partijen](https://github.com/VNG-Realisatie/api-beheer/blob/master/Deelnemende%20partijen.md) in te vullen. Ook de beheerder of beheerders van de standaard moeten na in beheername van een standaard of bij wijziging van de beheerder op deze lijst worden vermeldt.
 
+## Beheerproces
+Zodra een Open API standaard is ontwikkeld en goedgekeurd moet het worden overgedragen aan een beheerteam.
+Voor die betreffende standaard wordt dat beheerteam daarna verantwoordelijk voor:
+1. beantwoorden van vragen;
+2. het oplossen van problemen.
+
+Doorontwikkeling van de standaard hoort daar dus niet bij.
+
+*Beantwoorden van vragen*
+Voor het beantwoorden van vragen m.b.t. de betreffende standaard zijn twee kanalen beschikbaar. Ten eerste het specifiek voor de standaard beschikbaar gestelde kanaal in de Slack workspace 'VNG API Community' en ten tweede het eveneens specifiek voor de standaard beschikbaar gestelde discussie forum.
+
+_Beschouwen we de discussie voorzieningen op GitHub ook als een goed alternatief voor de discussie fora?
+Zo ja dan moeten we n.m.m. een keuze tussen beide maken. Het monitoren van meerdere omgevingen op vragen/wijzigingsverzoeken moet zo veel mogelijk voorkomen worden._
+
+Slack leent zich niet zo goed voor het gestructureerd voeren van een discussie aangezien in het kanaal meerdere onderwerpen door elkaar heen kunnen gaan lopen en (in de basis variant) historie slechts tot een x aantal reacties beperkt blijft. Slack is vooral geschikt voor het snel stellen en beantwoorden van enkelvoudige vragen.
+
+Voor het diepgaand bediscusieren van issues en ter sprake stellen van problemen in de standaard waarbij de discussies een lange(re) tijd beschikbaar moeten blijven kan beter het discussieforum worden gebruikt.
+
+Zodra het van belang is om een in Slack gevoerde discussie toch voor langere tijd te bewaren kunnen de beheerders de betreffende discussie alsnog onderbrengen op het discussie platform. Indien de discussie vragen over het gebruik en toepassing van de standaard betreft dienen de beheerders zich steeds af te vragen of het niet beter is de documentatie van de standaard zo aan te passen dat eenzelfde vraag in de toekomst voorkomen wordt.
+
+Vragen in beide kanalen moeten binnen xx dagen beantwoord zijn.
+
+*Probleemoplossing*
+De initiator voor probleemoplossing van een standaard is altijd een in GitHub ingebracht issue. 
+Indien het wijzigingsverzoek via Slack binnenkomt maakt de beheerder een issue aan in GitHub. 
```

Comment:
Ik zou het vragen aan diegene die het op Slack vraagt ipv zelf aan te maken. Zo voorkom je werk en dat wellicht informatie/kennis van de vrager verloren gaat.

Wij zeggen ook steeds tegen mensen: Maak maar een US aan ipv zelf iemands woorden in een US te gieten. De helft van de keren komt er geen US waarmee ook direct de prioriteit duidelijk is :)

Reply:
Helemaal mee eens. Daarnaast voed je gebruikers ook op om issues in te brengen.